packages/forma-36-react-components/src/components/ValidationMessage/ValidationMessage.css

```diff
@@ -4,12 +4,13 @@
 
 .ValidationMessage {
   display: flex;
-  align-items: center;
 }
 
 .ValidationMessage__icon {
   margin-right: calc(1rem * (7 / var(--font-base-default)));
-  margin-bottom: calc(1rem * (1 / var(--font-base-default)));
+  margin-top: calc(1rem * (2 / var(--font-base-default)));
```

Comment:
Nice @domarku 🔥do you want to use one of the spacing values here instead?

Reply:
100% yes, thanks for the suggestion

Reply:
Actually, come to think of it, when icons are put in context with text, it's good practice to use the base font size to set the relationship between the two. If the base font size changes, the chances are greater that it's still going to work, as opposed to using an external spacing variable. @m10l any thoughts?

Reply:
No strong opinion from me, think there are good arguments for both